### PR TITLE
Net udev rules

### PR DESCRIPTION
**Pull request by imobachgs**
Tests to check that [bsc#956605](https://bugzilla.suse.com/show_bug.cgi?id=956605). It relies on https://github.com/yast/autoyast-integration-test/pull/77 (included in aytests 1.0.2).
- `sles12.rb` includes now device renaming to check non-VNC installation with 2nd stage.
- `sles12_network.rb` test is added to check VNC installation with 2nd stage.
- `sles12_minimal.rb` is now performed through VNC and includes device renaming to check VNC installation without 2nd stage.

### CODE REVIEW

**Comment by ancorgs** [comment]
Adding VNC to sles12_minimal is not a little bit like hijacking the test? Wouldn't be better to have a separate one?

**Comment by imobachgs** [comment]
@ancorgs Yes, I see your point (I was also considering to add a new one). But I prefer to keep the number of tests low because each of them take a lot of time. I want to discuss with @schubi2 about `sles12_mini_network.rb` because, for me, that's the correct place.

**Comment by imobachgs** [comment]
Anyway, I've added a separate test (it was rather easy).

**Comment by ancorgs** [comment]
I just miss a comment somewhere saying that `udev_net_rules.sh` is not used in sles12_minimal because it does not work currently (although we have a PBI to make it work https://trello.com/c/SUdZnkzy/416-5-sp2-finish-moving-device-renaming-into-first-stage-in-case-of-ay-installation)

File: aytests/sles12_minimal.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 require "aytests/spec_helper"
 
-describe "SLES 12 checks," do
+describe "SLES 12 minimal," do
+  # udev network rules are not checked because they won't work
+  # in this scenario.
```
Comment (ancorgs) [inline aytests/sles12_minimal.rb:5]: I know it's not public (for no particular good reason), but what about linking here?
https://wiki.microfocus.net/index.php/YAST#Interface_renaming_problems_with_AutoYaST_in_SLE12-SP1

**Comment by imobachgs** [comment]
That's the reason to not link to that document. Maybe we should make that document public.

**Comment by ancorgs** [comment]
Ok, LGTM

**Comment by imobachgs** [comment]
Thanks!

**Pull request merged**